#include <infra/gen/gen-cpp2/volumeapi_types.h>
#include <infra/typestr.h>

template <>
const char* typeStr<volume::UpdateBlobMsg>()
{
    return "UpdateBlobMsg";
}

template <>
const char* typeStr<volume::UpdateBlobRespMsg>()
{
    return "UpdateBlobRespMsg";
}

template <>
const char* typeStr<volume::UpdateBlobMetaMsg>()
{
    return "UpdateBlobMetaMsg";
}

template <>
const char* typeStr<volume::UpdateBlobMetaRespMsg>()
{
    return "UpdateBlobMetaRespMsg";
}

